pyvo/dal/sia.py

```diff
@@ -503,12 +503,24 @@ def format(self):
 
     @format.setter
     def format(self, format_):
+        print("Set format")
         setattr(self, "_format", format_)
 
         if isinstance(format_, (str, bytes)):
```

Comment:
I think we can safely drop support for bytes here, while we are at it.  And I think what we should really be writing here is:

```
if not isinstance(format_, list):
    format_ = format_.split(",")
```

That way, we at least get less insane diagnostics if people pass in something else than lists or strings, and we support passing in list literals.